ScoutSuite/providers/aws/rules/findings/cloudformation-stack-with-role.json

```diff
@@ -1,5 +1,5 @@
 {
-    "description": "Role passed to stack",
+    "description": "Stack should not have a role",
     "rationale": "Passing a role to CloudFormation stacks may result in privilege escalation because IAM users with privileges within the CloudFormation scope implicitly inherit the stack's role's permissions.",
```

Comment:
You could instead amend the rationale:
```
Passing a role to CloudFormation stacks may result in privilege escalation because IAM users with privileges within the CloudFormation scope implicitly inherit the stack's role's permissions. Consequently, it should be ensured that the IAM privileges assigned to the stack's role follow the principle of least privilege.
```